e2e_tests/cli/test_import.py

```diff
@@ -78,15 +80,55 @@ def assert_same_annotation_properties(
             assert expected_property in actual_properties  # type : ignore
 
 
+def get_base_slot_of_item(
+    config_values: ConfigValues, dataset_id: int, item_idx: int
+) -> str:
+    """
+    Returns the base slot for the nth item in a specific `E2EDataset`. The base slot is
+    always the first listed slot
+    """
+    items = list_items(
+        config_values.api_key,
+        dataset_id,
+        config_values.team_slug,
+        config_values.server,
+    )
+    return items[item_idx]["slots"][0]["slot_name"]
```

Comment:
This might not be an issue now, but what if the BE decided to return the list of items in a different order? Wouldn't it be safer to find an item by item_id or name?